public/app/listeners/showLoadDatasetPrompt.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { dialog } = require('electron');
+const { RESPOND_LOAD_DATASET_PROMPT_CHANNEL } = require('../config/channels');
+const logger = require('../logger');
+
+const showLoadDatasetPrompt = (mainWindow) => (event, options) => {
```

Comment:
Here I would not have `options` as an argument, but as 
`const options = {
      filters: [{ name: 'json', extensions: ['json'] }],
    }; `

Reply:
👍 is this because we won't be uploading other types of files?